compiler/rustc_mir_transform/src/generator.rs

```diff
@@ -1314,6 +1317,8 @@ fn create_generator_resume_function<'tcx>(
     // unrelated code from the drop part of the function
     simplify::remove_dead_blocks(tcx, body);
 
+    AbortUnwindingCalls.run_pass(tcx, body);
```

Comment:
This should probably use `run_passes` too.

Reply:
Is there any benefit in calling `run_passes_no_validate` as opposed to just call to the pass directly?